frontend/packages/topology/src/actions/groupActions.ts

```diff
@@ -29,15 +30,15 @@ const cleanUpWorkloadNode = async (workload: OdcNodeModel): Promise<K8sResourceK
   return cleanUpWorkload(workload.resource, workload.data?.isKnativeResource ?? false);
 };
 
-const deleteGroup = (application: TopologyApplicationObject) => {
+const deleteGroup = (application: TopologyApplicationObject, t: TFunction) => {
```

Comment:
Interesting, you put `t` at the start of other methods 🤔 

I have no opinion one way or another, just thought it was noteworthy.